### ISSUE TITLE
Microsoft Teams and Visual Studio Code 

### ISSUE THREAD

**Issue by hasantahir** [godbout/Wooshy.docs#11]
Wow! Haven't been this excited with an app in a very very long time. Wanted to bring up issues on Microsoft Teams and Visual Studio Code where text is not recognised at all.

Cheers and eagerly following this.

PS: Is there an ELI5 for KindaVim? Not familiar with Vim, but wanted to get thoughts from you how to get up and running with no Vim experience.

**Comment by godbout** [comment]
> Wow! Haven't been this excited with an app in a very very long time. Wanted to bring up issues on Microsoft Teams and Visual Studio Code where text is not recognised at all.

will check, thanks.

> PS: Is there an ELI5 for KindaVim? Not familiar with Vim, but wanted to get thoughts from you how to get up and running with no Vim experience.

had no idea what a ELI5 was :D

kV is making all the hard choices for you so that you don't have to break your head. that being said, there's so many different apps and it's dealing with rough parts of the OS so you need sometimes to play with the `Families` in the Preferences and add your app there. apart from this, it's Vim. so best is to check for some Vim tutorial online. good luck! Vim motions are awesome.G

**Comment by godbout** [comment]
Microsoft Teams and Visual Studio Code supported in a3: https://github.com/godbout/Wooshy.docs/releases/tag/1a.3

**Comment by hasantahir** [comment]
Awesome; many thanks!
